src/components/Layout/index.test.tsx

```diff
@@ -1,12 +1,22 @@
 import { render, screen } from '@testing-library/react';
+import { createMemoryHistory } from 'history';
+import { Router } from 'react-router';
+
+import { StateContext } from 'contexts/UserContext';
+import { INITIAL_STATE } from 'contexts/UserContext/reducer';
 
 import Layout from '.';
 
 test('renders children without errors', () => {
```

Comment:
I know you didn't create this test, but I think we should always wrap our tests into a `describe` block.